packages/renderer/src/lib/image/ImageActions.svelte

```diff
@@ -96,6 +96,36 @@ function onError(error: string): void {
   icon="{faTrash}"
   enabled="{!image.inUse}" />
 
+{#if errorMessage}
+  <div class="modal fixed w-full h-full top-0 left-0 flex items-center justify-center p-8 lg:p-0 z-50" tabindex="-1">
+    <div class="border-t-red-600 border-t-2 p-4 bg-charcoal-600" aria-label="Success alert">
+      <div class="flex flex-row justify-center items-center pb-2">
+        <Fa icon="{faExclamationCircle}" class="text-red-500 mr-2" />
+        <div class="text-red-500 font-bold text-sm">
+          {errorTitle}
+        </div>
+        <Fa
+          icon="{faTimes}"
+          class="text-gray-900 pl-2 cursor-pointer"
+          on:click="{() => {
```

Comment:
Not related to this PR as it was already there but this doesn't work. The `on:click` on a Fa item is not triggered properly, at least on Windows. We should open an issue

Reply:
I can confirm the close icon does not work, I opened https://github.com/containers/podman-desktop/issues/5420